controllers/control/jobs.go

```diff
@@ -69,7 +69,7 @@ func (r *CassandraTaskReconciler) restartSts(ctx context.Context, sts []appsv1.S
 			}
 		}
 	}
-
+	restartedPods := make(map[string]int)
```

Comment:
Why do you need map[string]int for calculating int?

Reply:
You're right, the way you originally suggested doing it is correct I think. I've changed it back, no need for the map.